versions.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  required_version = ">= 0.12"
```

Comment:
Terraform 0.13 is probably going to be in a .0 release before the end of the summer. I don't think it would be a bad practice to constrain to `~> 0.12` until we see what the changes look like and if it could be a stumbling block for new users.